src/com/reason/ide/files/BsConfigJsonFile.java

```diff
@@ -0,0 +1,34 @@
+package com.reason.ide.files;
+
+import com.intellij.framework.detection.FileContentPattern;
+import com.intellij.lang.Language;
+import com.intellij.openapi.fileTypes.FileType;
+import com.intellij.openapi.vfs.VirtualFile;
+import com.intellij.patterns.ElementPattern;
+import com.intellij.psi.FileViewProvider;
+import com.intellij.util.indexing.FileContent;
+import com.intellij.util.indexing.FileContentImpl;
+import org.jetbrains.annotations.NotNull;
+
+public class BsConfigJsonFile extends FileBase {
```

Comment:
Avoid FileBase for anything that is not a module

Reply:
This change is stale. Fixed now.